_elements/form-controls.md

```diff
@@ -29,28 +29,28 @@ lead: Intro text on what is included in this section and how to use it. No more
   <input class="usa-input-focus" id="input-focus" name="input-focus" type="text">
 
   <div class="usa-input-error">
-    <label for="input-error">Text Input Error</label>
-    <span class="usa-input-error-message" role="alert">Helpful error message</span>
+    <legend>
```

Comment:
So that the error message would be part of the legend. I followed gov.uk's example: http://govuk-elements.herokuapp.com/errors

<img width="670" alt="screen shot 2015-09-08 at 3 45 16 pm" src="https://cloud.githubusercontent.com/assets/5249443/9749305/a6a9314c-5640-11e5-9011-dcb230d8655d.png">